user_guide_src/source/outgoing/view_layouts.rst

```diff
@@ -31,8 +31,21 @@ E.g. **app/Views/default.php**::
     </body>
     </html>
 
-The ``renderSection()`` method only has one argument - the name of the section. That way any child views know
-what to name the content section.
+The ``renderSection()`` method have two arguments ``$sectionName`` - the name of the section. That way any child views know
+what to name the content section. And ``$saveData`` - If true, saves data for subsequent calls, if false, cleans the data after displaying.
```

Comment:
I am not good at English, but these sentence should be one sentence?

Reply:
This is not coherent enough. Try this.

The ``renderSection()`` method has two arguments: ``$sectionName`` and ``$saveData``. ``$sectionName`` is the name of the section used by any child view to name the content section. If the boolean argument ``$saveData`` is set to true, the method saves data for subsequent calls. Otherwise, the method cleans the data after displaying the contents.

Reply:
Although, I'm not convinced for the explanation of `$sectionName`. It feels redundant.